application/common/config/main.php

```diff
@@ -95,15 +96,39 @@
                     },
                 ],
                 [
-                    'class' => EmailTarget::class,
-                    'categories' => ['application'], // stick to messages from this app, not all of Yii's built-in messaging.
-                    'logVars' => [], // no need for default stuff: http://www.yiiframework.com/doc-2.0/yii-log-target.html#$logVars-detail
+                    'class' => EmailServiceTarget::class,
```

Comment:
No, it wasn't. I simply copied/adapted this `EmailServiceTarget` configuration from [idp-pw-api's version](https://github.com/silinternational/idp-pw-api/blob/develop/application/common/config/main.php#L102-L144)... though it's interesting that that repo didn't limit by category. I'll add that back here.